config/config/common-config.yaml

```diff
@@ -704,6 +704,8 @@ calicoFelixMetrics:
   enabled: true
 
 clusterAdmin:
+  # The name of the user specified in /etc/kubernetes/admin.conf kubeconfig file of the control plane nodes
```

Comment:
I think this one should be moved to wc-config.yaml instead

Reply:
Sure, I moved it to wc-config.yaml under `gatekeeper.allowUserCRDs` since it is currently only used when user-CRDs are enabled. Let me know if you want it somewhere else, otherwise, it can be moved if this variable needs to be used for anything else in the future.